frida_tools/tracer.py

```diff
@@ -347,6 +347,22 @@ def _create_resolver_script(self):
     return workingSet;
 }
 
+function parseExportsFunctionPattern(pattern) {
+    var res = pattern.split('!');
+    var m, f;
+    if (res.length === 1) {
+        m = '*';
+        f = res[0];
+    } else {
+        m = (res[0] === '') ? '*' : res[0];
+        f = (res[1] === '') ? '*' : res[1];
+    }
+    return {
+        module: m,
+        function: f
+    };
+}
```

Comment:
Rebasing should get rid of this unrelated change.

Reply:
I'm not confident enough to know how to rebase these lines out, and I don't want to mess things up.  Can you rebase this for me (and explain what you did)?

Reply:
Reading up on rebasing, I moved to the "options_file_impl" branch and ran:

```
D:\Private\Dev\Open-Source-Projects\frida\frida-tools>git rebase master
Current branch options_file_impl is up to date.
```

`git status` shows no changes to `tracer.py`.  Please walk me through what commands I need to run to get the desired results.

Reply:
Do `git checkout master` and then `git pull`, so your local `master` is up-to-date with the upstream one. Then after that `git rebase`. (But make a backup first.)

Reply:
OK, did this. `git status` doesn't show any modifications to `tracer.py`, but I assume that the next time I commit/push the `options_file_impl` branch, it will fix itself.

Reply:
After that you can `git push -f` to overwrite the remote branch, which will update the PR.

Reply:
That was just what I needed <g>!

```
D:\Private\Dev\Open-Source-Projects\frida\frida-tools>git branch
  init_session_impl
  master
* options_file_impl
  user_data_impl

D:\Private\Dev\Open-Source-Projects\frida\frida-tools>git push -f
fatal: AggregateException encountered.
   One or more errors occurred.
Username for 'https://github.com': xxxxxxxxxx
Password for 'https://xxxxxxxxxx@github.com':
Everything up-to-date

D:\Private\Dev\Open-Source-Projects\frida\frida-tools>
```
If the remote was indeed updated, then the problem should be solved.  Please tell me if it is.

Reply:
Great, thank you! (Sorry for the delay here, I should be back on track soon 😅)